backend/routes/content.py

```
@@ -147,6 +147,12 @@ def get_module_content(module_id):
         elif module_type == ModuleType.BONUS_CHALLENGE:
             sub_dir = f"{order}_bonus_challenge"
             base_content_dir = os.path.join(base_content_dir, "bonus_challenges")
+        elif module_type == ModuleType.BONUS_SOLUTION:
+            sub_dir = f"{order}_bonus_solution"
+            base_content_dir = os.path.join(base_content_dir, "bonus_challenges")
+            logger.debug(
+                f"base_content_dir: {base_content_dir}, sub_dir: {sub_dir}, order: {order}"
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)